### PR TITLE
Drop unused columns left over from MongoDB migration

### LINKED ISSUE #8828: Drop unused columns left over from MongoDB migration
There are currently five completely unused columns that are left over from a long-gone migration from MongoDB:

https://github.com/PrairieLearn/PrairieLearn/blob/29ddbbf3ac4336630fb068232902a2539f7b81e1/apps/prairielearn/src/migrations/20170428200859_initial_state.sql#L316-L318

https://github.com/PrairieLearn/PrairieLearn/blob/29ddbbf3ac4336630fb068232902a2539f7b81e1/apps/prairielearn/src/migrations/20170428200859_initial_state.sql#L361

https://github.com/PrairieLearn/PrairieLearn/blob/29ddbbf3ac4336630fb068232902a2539f7b81e1/apps/prairielearn/src/migrations/20170428200859_initial_state.sql#L377

We should drop these columns from the database. Note that `submissions.sid` cannot be safely removed until #8827 is deployed.

### PR DESCRIPTION
**Pull request by nwalters512**
Closes #8828.

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -31,7 +31,7 @@ start-executor:
 	@node apps/prairielearn/dist/executor.js
 
 update-database-description:
-	@yarn --cwd apps/prairielearn pg-describe postgres -o ../../database
+	cd apps/prairielearn && yarn pg-describe postgres -o ../../database
```
Comment (jonatanschroeder) [inline Makefile:34]: Any reason for this change?

**Review by jonatanschroeder — APPROVED** [review]

File: Makefile (hunk 1 above)
Comment (nwalters512) [reply Makefile:34]: For some reason Yarn wasn't picking up the `pg-describe` executable with the `yarn --cwd ...` version. I didn't spend too much time figuring out why. I'm curious, do you see similar behavior locally, or is this something with my environment?

Comment (jonatanschroeder) [reply Makefile:34]: This works for me with the `--cwd` option. Another option that works for me is:
```
yarn workspace @prairielearn/prairielearn pg-describe postgres -o ../../database
```

Comment (nwalters512) [reply Makefile:34]: That works for me now, huh! I kinda like that better, let me use that instead.

**Review by jonatanschroeder — APPROVED** [review]

**Pull request merged**